module/Calculator/ColorJudge.cpp

```diff
@@ -0,0 +1,126 @@
+/**
+ * @file ColorJudge.cpp
+ * @brief 色識別クラス
+ * @author keiya121
+ */
+
+#include "ColorJudge.h"
+
+// RGBから色を判別する
+COLOR
+ColorJudge::getColor(rgb_raw_t const& _rgb)
```

Comment:
変な所で改行されてる。コード整形してもこのままだったのかな？
```suggestion
COLOR ColorJudge::getColor(rgb_raw_t const& _rgb)
```

Reply:
ここについては、自動整形もCI/CDも改行ありなし両方がclang-format通ってなかったです。
改行なしの方向で修正し、プッシュしました。